src/Controller/UserController.php

```diff
@@ -812,17 +814,17 @@ public function setProjectManagerForUser(
             return new JsonResponse(
                 [
                     'code' => Response::HTTP_NOT_FOUND,
-                    'message' => 'Not found!'
+                    'message' => 'Project Manager to assign not found!'
                 ],
                 Response::HTTP_NOT_FOUND
             );
         }
-
-        if ($projectManager->getRoles() === array('ROLE_USER')) {
+        $projectManagerRole = $projectManager->getRoles();
+        if ($projectManagerRole !== array('ROLE_PM') || $projectManagerRole !== array('ROLE_ADMIN')) {
```

Comment:
You should be using a specific Symfony function to check the role. As ROLE_ADMIN ha already ROLE_PM you won't have to specify it.